START.md

```diff
@@ -258,6 +301,13 @@ Usage of ./AutoSpotting:
 
   -tag_filters=[{spot-enabled true}]: Set of tags to filter the ASGs on.  Default is -tag_filters 'spot-enabled=true'
         Example: ./AutoSpotting -tag_filters 'spot-enabled=true,Environment=dev,Team=vision'
+
+  -beanstalk_cfn_wrappers="on":
```

Comment:
I would propose to rename this to something like `patch_beanstalk_userdata`, and then we can also keep the true/false boolean values